pkg/controller/statefulset/stateful_set_control.go

```diff
@@ -602,9 +610,28 @@ func (ssc *defaultStatefulSetControl) updateStatefulSet(
 		}
 	}
 
+	// refresh states for all pods
+	requeueDuration := requeueduration.Duration{}
```

Comment:
There is no need to new `requeueDuration` here, for the `duration` returned below can directly push into durationStore.